src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
+    let { _simplaObservers: observers } = this,
+        simplaLogin = this.$['login'],
         promptLogin = (editable) => {
           if (editable && !this._authenticated) {
-            simplaLogin.prompt()
-              .then(loggedIn => Simpla.editable(loggedIn));
+            simplaLogin.prompt().then(loggedIn => {
+              Simpla.editable(loggedIn)
+            });
           }
         },
         initialPrompt = () => {
+          let shouldPrompt = Simpla.getState('editable') && loginPrompt;
+
           if (typeof simplaLogin.prompt === 'function') {
-            promptLogin(Simpla.getState('editable'));
+            promptLogin(shouldPrompt);
```

Comment:
What? It's a recursive function, it's exactly the same as `setInterval`, just more concise.

I didn't think `simplaLogin.prompt` checking should be in `promptLogin`, because that's lumping that extra logic into something that doesn't need it most of the time - basically you're just saying combine these two functions, I thought it was good to split them up because they're doing different things - one is prompting for login with a true/false param, the other is checking if it's possible to prompt for login then prompting for login if so.